main.go

```diff
@@ -221,7 +228,34 @@ func (p Proxy) getSite(level string) (string, error) {
 	return upstream, nil
 }
 
-func getClaimFromToken(secret []byte, token string) (ProxyClaim, error) {
+func (p Proxy) clearQsToken(r *http.Request) {
```

Comment:
Could we rename this to `clearQueryToken` or `clearTokenFromQuery`?


Or merge it with clearFlag, using something like `clearQueryParam(request, param)`. I prefer this way unless clearFlag should not be modifying redirect.